app/views/courses/show.html.erb

```diff
@@ -1,17 +1,50 @@
+<script>
+function drawFlashcard() {
+  var xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+    console.log(this.status);
+    if (this.readyState == 4 && this.status == 200) {
```

Comment:
Out of curiosity, what do 4 and 200 do/mean in this context?

Reply:
4 means "AJAX response received" and 200 is the HTML status code for "no errors, returning requested information"